suite-native/module-connect-device/src/components/ConnectingTrezorHelp.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+
+import { BottomSheet, IconButton, Text, VStack } from '@suite-native/atoms';
+import { Translation, useTranslate } from '@suite-native/intl';
+
+export const ConnectingTrezorHelp = () => {
```

Comment:
Rename this please so the filename has suffix `Button` e.g `ConnectTrezorHelpButton`. Because it does render the button in the first place. So it follows the same convention es [AccountRenameButton](https://github.com/trezor/trezor-suite/blob/b0158c70030b73939fb68d074720701c13ccf42c/suite-native/module-accounts-management/src/components/AccountRenameButton.tsx) does for example.